### PR TITLE
fix(comment): Update of empty comment not allowed.

### PR DESCRIPTION
**Pull request by vikram-raj**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [ ] What does this PR do? 
Update comment with empty string not allowed.

- [ ] What issue/task does this PR references?
https://github.com/openshiftio/openshift.io/issues/2538

- [ ] Are the tests Included?

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @vikram-raj for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
[test]

**Comment by vikram-raj** [comment]
> @vikram-raj now I can't update an empty comment but I can reply with an empty comment.

@Raunak1203 Fix this issue but this fix is dependent on this PR https://github.com/fabric8-ui/ngx-widgets/pull/147

**Comment by vikram-raj** [comment]
[test]

File: src/app/widgets/comment-module/comment.component.ts [hunk 1]
```diff
@@ -57,16 +60,42 @@ export class CommentComponent {
...
+    let inputText = this.replyEditor.editorInput.nativeElement.innerText.trim();
+    if (inputText.length) {
+      this.replyEditor.saving = false;
+    } else {
+      this.replyEditor.saving = true;
+    }
+  }
+
+  onReplyClick() {
+    this.replyActive=true;
```
Comment (sudsen) [inline src/app/widgets/comment-module/comment.component.ts]: Space is missing in both side of the `=`

**Comment by sudsen** [comment]
I don't think this fix should be in the planner. This is a `markdown-component` fix.

**Comment by sanbornsen** [comment]
@vikram-raj so what do you think? Can we live with the changes in `ngx-widgets` and use the param here?

**Comment by vikram-raj** [comment]
@sanbornsen I am fine with the changes in ```ngx-widgets``` and using the param in planner.

**Comment by vikram-raj** [comment]
This issue has been fixed at the component level. Hence closing this and raised a new PR https://github.com/fabric8-ui/fabric8-planner/pull/2681.

**Pull request closed without merging**